br/pkg/task/backup.go

```diff
@@ -343,7 +343,7 @@ func RunBackup(c context.Context, g glue.Glue, cmdName string, cfg *BackupConfig
 	})
 
 	// nothing to backup
-	if ranges == nil {
+	if ranges == nil || len(ranges) <= 0 {
```

Comment:
```suggestion
	if len(ranges) == 0 {
```

`len(nil) == 0` and I guess `len(anything)` should not return negative numbers?